src/dds/dds-device-watcher.cpp

```diff
@@ -3,62 +3,63 @@
 
 #include <types.h>
 #include "dds-device-watcher.h"
-#include "msg/devicesPubSubTypes.h"
+#include <librealsense2/dds/topics/dds-messages.h>
 #include <fastdds/dds/domain/DomainParticipantFactory.hpp>
 #include <fastdds/dds/domain/DomainParticipant.hpp>
 #include <fastdds/dds/subscriber/qos/DataReaderQos.hpp>
 #include <fastdds/dds/subscriber/DataReader.hpp>
 #include <fastdds/dds/subscriber/Subscriber.hpp>
 
-// We align the DDS topic name to ROS2 as it expect the 'rt/' prefix for the topic name
-#define ROS2_PREFIX( name ) std::string( "rt/" ).append( name )
-
 using namespace eprosima::fastdds::dds;
-
 using namespace librealsense;
 
 dds_device_watcher::dds_device_watcher( int domain_id )
     : _participant( nullptr )
     , _subscriber( nullptr )
     , _topic( nullptr )
     , _reader( nullptr )
-    , _type_ptr( new devicesPubSubType() )
+    , _type_ptr( new dds::topics::raw::device_infoPubSubType() )
```

Comment:
Hmm I don't like this... let's talk?

Reply:
Let's change `_type_ptr` to `_topic_type`?

Reply:
```
class device_info
{
    using type = raw::device_infoPubSubType;
};
```
Or a `typedef`...
And for the server, too.

Reply:
Done